rump/Manhattan_Plot.rmd

```diff
@@ -0,0 +1,146 @@
+---
+title: "Manhattan Plot"
+author: "Yasmine Gillespie"
+date: "6/6/2020"
+output: html_document
+---
+
+
+```{r , warning=FALSE, include=FALSE}         

```

Comment:
Great job! Please remove line 9-11, since those packages are assumed to be already installed by others (i.e., they will be added to Docker container)